plugins/linux-capture/pipewire.c

```diff
@@ -472,6 +488,11 @@ static void on_process_cb(void *user_data)
 			     obs_pw->format.info.raw.format);
 			goto read_metadata;
 		}
+		if ((buffer->datas[0].chunk->flags & SPA_CHUNK_FLAG_CORRUPTED) >
+		    0) {
+			blog(LOG_ERROR, "[pipewire] buffer plane corrupt");
```

Comment:
Should this be log_debug like the other reports?

Reply:
Oh thanks, missed to change that back.